examples/oft-solana/tasks/common/taskHelper.ts

```diff
@@ -0,0 +1,56 @@
+import { OmniAddress } from '@layerzerolabs/devtools'
+import { EndpointId } from '@layerzerolabs/lz-definitions'
+import { UlnProgram } from '@layerzerolabs/lz-solana-sdk-v2'
+import {
+    IEndpointV2,
+    Timeout,
+    Uln302ConfigType,
+    Uln302ExecutorConfig,
+    Uln302UlnConfig,
+} from '@layerzerolabs/protocol-devtools'
+
+/**
+ * Get the receive config for a Solana OApp
+ * @param endpointV2Sdk {IEndpointV2} SDK for the endpoint
+ * @param remoteEid {EndpointId} remote eid
+ * @param address {OmniAddress} address of the OApp
+ */
+export async function getSolanaReceiveConfig(
+    endpointV2Sdk: IEndpointV2,
+    remoteEid: EndpointId,
+    address: OmniAddress
+): Promise<[OmniAddress, Uln302UlnConfig, Timeout] | undefined> {
+    const [receiveLibrary] = await endpointV2Sdk.getReceiveLibrary(address, remoteEid)
+    return [
+        receiveLibrary ?? UlnProgram.PROGRAM_ADDRESS,
+        await endpointV2Sdk.getAppUlnConfig(
+            address,
+            UlnProgram.PROGRAM_ID.toBase58(),
+            remoteEid,
+            Uln302ConfigType.Receive
+        ),
```

Comment:
do you think this is worth putting in a variable above and also in a try-catch? Same question for usages below